README.md

```diff
@@ -1,32 +1,50 @@
 # plug
 PortabLe infrastrUcture for Grid modeling
 
+### Prerequisites
+To use *plug* you'll need to have docker installed - visit their [website](https://docs.docker.com/get-docker/)
+for details.
+
+If you want to run simulations using the Gurobi solver (recommended), you'll need a
+license file called `gurobi.lic` and located in the `gurobi_license` folder as shown below.
+
+*NOTE*: certain license types are not supported at the moment. 
 
-### How to use
-To run the containerized framework, the only prerequisite is to have a gurobi
-license called `gurobi.lic` in the `gurobi_license` folder as shown below. The required
-images will be downloaded automatically from our [container registry](https://github.com/orgs/Breakthrough-Energy/packages).
 
-Optionally, if you want to build any of the docker images locally (e.g. for development purposes),
+**Optional**: if you want to build any of the docker images locally (e.g. for development purposes),
 you should use the following directory structure so relative paths will work.
 
 ```bash
     .
     ├── plug
         ├── gurobi_license/gurobi.lic
     ├── PowerSimData
+    ├── PostREISE
     └── REISE.jl
 ```
 
+### How to use
+
 We describe the workflow for running a standalone installation on a single
 computer. To get started, run `cd standalone` in your shell, followed by
-`docker-compose up`. In a separate shell, attach to the powersimdata client using
+`docker-compose up` (optionally pass `-d` to run in the background). 
+The docker images will be downloaded automatically 
+from our [container registry](https://github.com/orgs/Breakthrough-Energy/packages).
+
+The `client` container contains both [PowerSimData] and [PostREISE] packages, for
+scenario management and analysis, respectively. The default compose file starts the
+client running `bash`, which serves as an entrypoint to either an `ipython` shell:
 
 ```
-docker-compose exec powersimdata ipython
+docker-compose exec client ipython
 ```
+ or a jupyter notebook:
 
-See the PowerSimData [readme](https://github.com/Breakthrough-Energy/PowerSimData) for details 
+```
+docker-compose exec client jupyter lab --port=10000 --no-browser --ip=0.0.0.0 --allow-root
```

Comment:
Since this command is already in the PostREISE docker file, I think we just need to run:
`docker-compose up`
Maybe we could mention this and then later have this line as a more explicit option if they want to change the defaults. The nice thing about Jupyter Lab is that you can use notebooks, the python REPL, and the terminal within the same IDE environment.

Reply:
So there are 2 compose files now - in the `training.yml` I left the jupyter entrypoint and will be adding some docs where we use that default. The original compose file I left with `bash` so it's an explicit choice. TBH it's mostly easier for me to develop with for now, but I'm open to changing it later.